src/libraries/sendtohelixhelp.proj

```diff
@@ -240,9 +240,23 @@
     </ItemGroup>
   </Target>
 
+  <!-- XUnitLogChecker required configuration -->
   <ItemGroup Condition="Exists('$(XUnitLogCheckerLibrariesOutDir)')">
     <HelixCorrelationPayload Include="$(XUnitLogCheckerLibrariesOutDir)" />
-  </ItemGroup>  
+    <HelixCorrelationPayload Condition="'$(WindowsShell)' == 'true'" Include="dotnet-sos">
+      <Destination>sos</Destination>
+      <Uri>https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/flat2/dotnet-sos/$(DotnetSosVersion)/dotnet-sos.$(DotnetSosVersion).nupkg</Uri>
+    </HelixCorrelationPayload>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetOS)' == 'windows'">
+    <SosPreCommand Include="set _NT_SYMBOL_PATH=%HELIX_CORRELATION_PAYLOAD%;%HELIX_CORRELATION_PAYLOAD%\PDB;%HELIX_CORRELATION_PAYLOAD%\shared\$(MicrosoftNetCoreAppFrameworkName)\$(ProductVersion)" />
```

Comment:
you can't use semicolons in cases like this. may need to be a property and add it at the end. Also, escape setting _NT_SYMBOL_PATH with `%22`

Reply:
> may need to be a property and add it at the end

I don't understand that suggestion. Would you mind elaborating? Why wouldn't `;` work as an env var value separator?

> escape setting _NT_SYMBOL_PATH with %22

Not sure I follow. Did you actually mean escaping the `%` character I used for `%HELIX_CORRELATION_PAYLOAD%`? If that's the case, I am looking at other examples in the same file, and when an env var is set in an ItemGroup Include attribute, there's no need for escaping (I have no idea why). Escaping seems to only be required when setting an env var value in the xml value of a property (raw text).

Reply:
Ok I see. The semicolon is not going to concatenate the values of the _NT_SYMBOL_PATH env var. Instead, they are getting printed in separate lines. I'm trying an alternative approach.

Reply:
Pushed a commit that ensures the env var values separated by `;` get printed in the same line, and I avoided using an item group for that. I tested it locally and it worked. Can you please take another look, @hoyosjs?